frontend/src/metabase/dashboard/components/Dashboard/use-dashboard-scroll.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+
+import { getMainElement } from "metabase/lib/dom";
+
+export const useDashboardScroll = ({
+  isInitialized,
+  isSdk,
```

Comment:
I didn't see `isSdk` referenced in this PR, is this meant to be use in the future?